simplified-ui-accounts/src/main/java/org/nypl/simplified/ui/accounts/AccountListRegistryViewModel.kt

```diff
@@ -48,6 +48,11 @@ class AccountListRegistryViewModel(private val locationManager: LocationManager)
   private val logger =
     LoggerFactory.getLogger(AccountListRegistryViewModel::class.java)
 
+  private val featuredLibrariesList = arrayOf(
```

Comment:
Instead of having this here, I think this list should be defined in [PalaceBuildConfigurationService](https://github.com/ThePalaceProject/android-core/blob/main/simplified-app-palace/src/main/java/org/thepalaceproject/palace/PalaceBuildConfigurationService.kt), where we have other "configuration" settings for the app. It will be easier to find if we want to change it.

Reply:
I don't know how I've missed the Notion task description suggesting that file to be the one with this list. I'll move it.